### DISCUSSION TITLE
Change the default Panel on Select Element

### DISCUSSION THREAD

**Discussion by LamaYouzghatli** [GrapesJS/grapesjs#3967] [Q&A] [answered]
Hi,

I want to show trait manager panel on selecting element instead of style manager.

How can I do that?

**Comment by ronaldohoch** [comment, marked as answer]
Hello, i think you're using the grapesjs-preset-webpage plugin, if so, here is the code that select and show style manager, try to adapt it.

https://github.com/artf/grapesjs-preset-webpage/blob/master/src/panels/index.js#L109

**Reply by LamaYouzghatli** [reply]
Yes! that's it. Thanks a lot.
